assignments/objects.js

```diff
@@ -18,47 +18,137 @@ const example = {
 }
 
 // Write your intern objects here:
-
+internMitzi = { "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"}
+internKennan = { "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"}
+internKeven = { "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"}
+internGannie = { "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"}
+internAntonietta = { "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"}
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(internMitzi.name);
 // Kennan's ID
-
+console.log(internKennan.id);
 // Keven's email
-
+console.log(internKeven.email);
 // Gannie's name
-
+console.log(internGannie.name);
 // Antonietta's Gender
-
+console.log(internAntonietta.gender);
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+internKennan.speak = function(){
+  return `Hello, my name is ${this.name}!`;
+}
+console.log(internKennan.speak());
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
+internAntonietta.multiplyNums = function(x,y){
+  return x * y;
+}
+console.log(internAntonietta.multiplyNums(3,4));
 
 // === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
 
 // ==== Stretch Challenge: Nested Objects and the this keyword ==== 
 
 // 1. Create a parent object with properties for name and age.  Make the name Susan and the age 70.
+//const parent = {};
+const parent = {"name":"Susan","age":70};
 // 2. Nest a child object in the parent object with name and age as well.  The name will be George and the age will be 50.
+parent.child = {"name":"George","age":50};
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
+parent.child.grandchild = {"name":"Sam","age":30};
```

Comment:
Awesome use of assignation.  Other coders might fight the object written out literally easier to read.